### DISCUSSION TITLE
Is there a way to update the CSS, HTML and JS only for the front end web page?

### DISCUSSION THREAD

**Discussion by grexzen** [rom1504/clip-retrieval#222] [Q&A] [answered]
Thank you so much from your framework!

I have learned so much by just doing, and following your lead.

I have been able to replicate everything here to embed my gens, and use your webpage to filter, display, and search them.

I was wondering if there is a way to modify the front end look and feel on my end?

Or do I need to install this on my repository, edit those files there, then start the web server from there? 

Thanks again!

**Comment by rom1504** [comment, marked as answer]
hey, yeah you can take only this dir https://github.com/rom1504/clip-retrieval/tree/main/front and follow installation for it

**Reply by grexzen** [reply]
Thanks so much!
